server/xpub-client/index.js

```diff
@@ -1,7 +1 @@
-const S3Storage = require('./s3Storage')
-const ScienceBeamApi = require('./scienceBeamApi')
-
-module.exports = {
-  S3Storage,
-  ScienceBeamApi,
-}
+module.exports = {}
```

Comment:
This can be deleted and the package file / config for registering `xpub-client` can also be removed